.github/workflows/macos.yaml

```diff
@@ -13,10 +13,12 @@ on:
     branches:
     - master
     - macOSRunner
+    - ErrnoNotEPIPEServerTest
```

Comment:
Please remove these before merging, and the macOSRunner one too

Reply:
Done